CMakeLists.txt

```diff
@@ -207,6 +221,12 @@ if(APPLE)
     endif()
 endif()
 
+if(OSXCROSS_TARGET)
+    set(CMAKE_C_FLAGS "-I${OSXCROSS_SDK}/usr/include ${CMAKE_C_FLAGS}")
```

Comment:
Could use `list([APPEND](https://cmake.org/cmake/help/latest/command/list.html#append) <list> [<element>...])` syntax here but not really a big deal

Reply:
I tried using the following but I get build errors.
```cmake
list(PREPEND CMAKE_C_FLAGS -I${OSXCROSS_SDK}/usr/include)
list(PREPEND CMAKE_CXX_FLAGS -I${OSXCROSS_SDK}/usr/include)
list(APPEND CMAKE_SHARED_LINKER_FLAGS -L${CMAKE_OSX_SYSROOT}/usr/lib)
```
Maybe I'm missing something?

Reply:
Hmm I think we had issues with this before. I'm not exactly sure why. Maybe just leave it. Tbh we shouldn't be setting `CMAKE_C_FLAGS ` or similar flags directly. It should be set on the target and CMake will populate all of these for us. We can worry about that later.